extensions/ql-vscode/test/vscode-tests/cli-integration/jest.setup.ts

```diff
@@ -8,23 +8,24 @@ import { dirname } from "path";
 import fetch from "node-fetch";
 import { DB_URL, dbLoc, setStoragePath, storagePath } from "./global.helper";
 import * as tmp from "tmp";
-import { getTestSetting } from "../test-config";
 import { CUSTOM_CODEQL_PATH_SETTING } from "../../../src/config";
-import { extensions, workspace } from "vscode";
+import { ConfigurationTarget, env, extensions, workspace } from "vscode";
+import { beforeEachAction } from "../test-config";
 
-import baseJestSetup from "../jest.setup";
-
-export default baseJestSetup;
+(env as any).openExternal = () => {
+  /**/
+};
```

Comment:
Should this be mocked instead of outright overwritten?

Reply:
We can definitely mock this instead, that will make it easier to test that any calls to this function are being made. This was a copy of the previous implementation, but I'll also change that to be a mocked function.